### PR TITLE
Improvements work 2

### LINKED ISSUE #1168: [SUGGESTION] | Blood knights holy order
**Describe your suggestion in full detail below:**
Sunwell religion kinda lacks holy orders, but it had one, blood knights. They were created after Sunwell fallen and elfs became Sin'Dorei, but I think decision must be added even before Sunwell got corrupted, but with higher price than after corruption. I suggest like 5k pretige, 1k gold, 1k piety and controlling 4-5 holy places. For Sin'Dorei 1k prestige 500 gold and piety with controlling 3-4 holy places. And maybe adding paladins to blood knights will be fine idea, but idk.

### LINKED ISSUE #1269: Unite Elven Cultures in One Group
**Describe your suggestion in full detail below:**
All kinds of elves (satyrs, high elves, night elves, naga) speak very close languages, so I suggest uniting them in one Elven group.

### LINKED ISSUE #1305: One Sunwell to convert them (elves) all
**Describe your suggestion in full detail below:**
Allow the Sunwell to convert not only the high elves but all the elves (including satyrs, naga).
<details>
<summary>Click to expand</summary>

image

</details>

### LINKED ISSUE #1533: High elven temple icons
**Describe your suggestion in full detail below:**
You wouldn't believe me. I found out we have holding icons for high elven temples in `Warcraft-Guardians-of-Azeroth\gfx\interface\holdings\highelfgfx`. So, you know, we can use them.

### PR DESCRIPTION
**Pull request by BurnedCrusader**
## Changelog:
- Changed high elf temple icons.
- Moved all the elves to one culture group (needs balance testing).
- Created Blood Knights as religious order and allowed to create it for heavy price as high elf and small as blood elf.
- Allowed Sunwell to convert all the elves.

## Developer changelog:
- The new culture group needs testing for balance.
- Blood Knights need testing too for any hole in code.
- Added and changed a lot of localization, so needs a translation to german and french sooner or later.

# How to test:
Play as Anasterian or Lor'Themar(in 605 start date)

Closes #1533 Closes #1305 Closes #1269 Closes #1168

### CODE REVIEW

File: common/buildings/wc_tradepost_culture.txt [hunk 1]
```diff
@@ -48,7 +48,10 @@ trade_post = {
 	tr_culture_group_night_elf = {
 		potential = {
 			in_trade_route_trigger = yes
-			FROM = { culture_group = night_elf_group }
+			FROM = { 
+				culture = night_elf
+				culture = highborne 
+			}
```
Comment (zumbak04) [inline common/buildings/wc_tradepost_culture.txt:54]: You forgot OR, you can't have 2 cultures at the same time.
```suggestion
			FROM = {
				OR = {
					culture = night_elf
					culture = highborne 
				}
			}
```

File: interface/wc_decision_icons.gfx [hunk 2]
```diff
@@ -348,4 +348,8 @@ spriteTypes = {
 		name = "GFX_repay_money"
 		texturefile = "gfx\\interface\\decision_icons\\repay_action.tga"
 	}
+	spriteType = {
+		name = "GFX_create_blood_knights_order"
+		texturefile = "gfx\\interface\\decision_icons\\restore_the_sunwell.dds"
```
Comment (zumbak04) [inline interface/wc_decision_icons.gfx:353]: Can we get a new icon?
<details>
<summary>Click to expand</summary>

image
image

</details>

File: decisions/wc_order_decisions.txt [hunk 3]
```diff
@@ -250,8 +250,54 @@ decisions = {
...
+		}
+		allow = {
+		    trigger_if = {
+                limit = { has_global_flag = blood_elves_happened_flag }
+				num_of_holy_sites = 3
+				piety = 250
+			}
+            trigger_else = { 
+				num_of_holy_sites = 4
+				piety = 2000
```
Comment (zumbak04) [inline decisions/wc_order_decisions.txt:289]: That's A LOT of piety. Even reforming religion requires 750.

File: events/wc_order_events.txt [hunk 4]
```diff
@@ -906,31 +906,130 @@ narrative_event = {
...
+			religion = cult_of_sunwell
+			culture = high_elf
+			age = 300
+			trait = creature_high_elf
+			trait = class_paladin_6
+			trait = brilliant_strategist
+		}
+		new_character = {
+		    set_defacto_liege = ROOT
+		}
```
Comment (zumbak04) [inline events/wc_order_events.txt:983]: So, these paladins end up in Quel'thalas court, is it intended?
<details>
<summary>Click to expand</summary>

image

</details>

File: decisions/wc_order_decisions.txt (hunk 3 above)
Comment (MemeWizard) [reply decisions/wc_order_decisions.txt:289]: If anything, forming a holy order for eastern religions in vanilla require 1000.

File: events/wc_order_events.txt [hunk 5]
```diff
@@ -906,31 +906,130 @@ narrative_event = {
...
+	}
+	show_root = yes
+	
+	immediate = {
+		create_character = {
+			random_traits = yes
+			dynasty = none
+			female = no
+			religion = cult_of_sunwell
+			culture = high_elf
```
Comment (zumbak04) [inline events/wc_order_events.txt:932]: I'm Sin'dorei, the order leader is Quel'dorei.
<details>
<summary>Click to expand</summary>

image
image

</details>

```suggestion
			culture = ROOT
```

File: events/wc_order_events.txt [hunk 6]
```diff
@@ -906,31 +906,130 @@ narrative_event = {
...
 	}
 }
+# The Creation of Blood Knights Order
+namespace = WCBKO
+narrative_event = {
+	id = WCBKO.01
+	title = EVTTITLE_WCBKO_1
+	desc = EVTDESC_WCBKO_1
+	picture = GFX_evt_sunwell
+	border = GFX_event_narrative_frame_war
```
Comment (zumbak04) [inline events/wc_order_events.txt:916]: Show the order's leader portrait in the event. This should work.
```suggestion
	border = GFX_event_narrative_frame_war
	portrait = event_target:target_order_leader
```

File: events/wc_order_events.txt (hunk 4 above)
Comment (BurnedCrusader) [reply events/wc_order_events.txt:983]: Kinda yes, Kinda no

File: interface/wc_decision_icons.gfx (hunk 2 above)
Comment (BurnedCrusader) [reply interface/wc_decision_icons.gfx:353]: Will try to find it then and make it .dds file

File: events/wc_order_events.txt [hunk 7]
```diff
@@ -906,31 +906,130 @@ narrative_event = {
 		name = EVTOPTB_WCORD_406
 	}
 }
+# The Creation of Blood Knights Order
+namespace = WCBKO
+narrative_event = {
+	id = WCBKO.01
```
[the remark this answers is not in the document]
Comment (BurnedCrusader) [reply events/wc_order_events.txt:912]: > Can we not use 2 namespaces in one file?
Where? There's only one that I see(or i'm blind)

> There's no need in 01, the event ID number will be 1 anyway.
Okay, will have that in mind

File: events/wc_order_events.txt (hunk 4 above)
Comment (BurnedCrusader) [reply events/wc_order_events.txt:984]: I wanted them to be in order court, but they appeared in Quel'Thalas court, so I thought _this is kinda logical and I won't touch it them_

File: events/wc_order_events.txt (hunk 7 above)
[the remark this answers is not in the document]
Comment (zumbak04) [reply events/wc_order_events.txt:912]: > Where? There's only one that I see(or i'm blind)

Your WCBKO and WCORD.

[the remark this answers is not in the document]
Comment (BurnedCrusader) [reply events/wc_order_events.txt]: Done

File: decisions/wc_order_decisions.txt (hunk 3 above)
Comment (BurnedCrusader) [reply decisions/wc_order_decisions.txt]: Changed back to 1000

File: localisation/00_zuwc_united_localisation.csv [hunk 8]
```diff
@@ -2848,10 +2848,10 @@ GetMonstrousUndeadNameDesc_Abomination;a horrific creation that is sewn together
...
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+EVTTITLE_WCORD_74;Creation of Blood Knights Order;;;;;;;;;;;;;x
+EVTDESC_WCORD_74;After many centuries of fighting against trolls and others high elfs established and craved from bones of their enemies new Empire for all world to envy. Quel'Thalas, eversummer land, now is threatened by new threats. Trolls seeking vengeance for their bretheren, vile undead trying to complete their master's plan, evil demons trying to destroy Azeroth and awakened Old Gods. All trying to destroy new home of elfs, even xenophobic humans want to see Quel'Thalas burn. No longer able to wait [From.GetFirstName] created Blood Knights Order, Order of elven paladins that will help defend Quel'Thalas and Sunwell against any outside threat. All around realm paladins bring their weapons and faith to Silvermoon city and join Blood Knights ranks. Light shines above Silvermoon city.;;;;;;;;;;;;;x
+nick_the_sunblade;the Sunblade;;;;;;;;;;;;;x
+EVTOPTA_WCORD_74;May light be with them;;;;;;;;;;;;;x
```
Comment (arithon) [inline localisation/00_zuwc_united_localisation.csv]: Did you write this line on the 4th of May? :D

File: localisation/00_wc_job_titles.csv [hunk 9]
```diff
@@ -218,22 +218,6 @@ job_treasurer_murloc_group;Mmmml Mmgr;Mmmml Mmgr;Mmmml Mmgr;;Mmmml Mmgr;;;;;;;;;
...
-job_spymaster_naga_group;Triton;;Triton;;;;;;;;;;;x
-job_spymaster_female_naga_group;Siren;;Sirene;;;;;;;;;;;x
-job_chancellor_night_elf_group;Emissary;;Botschafter;;;;;;;;;;;x
-job_chancellor_female_night_elf_group;Emissary;;Botschafterin;;;;;;;;;;;x
-job_marshal_night_elf_group;General;G�n�ral;General;;;;;;;;;;;x
-job_marshal_female_night_elf_group;General;G�n�ral;Gener�lin;;;;;;;;;;;x
-job_treasurer_night_elf_group;Quartermaster;;Quartiermeister;;;;;;;;;;;x
-job_treasurer_female_night_elf_group;Quartermaster;;Quartiermeisterin;;;;;;;;;;;x
-job_spymaster_night_elf_group;Nightblade;;Nachtklinge;;;;;;;;;;;x
-job_spymaster_female_night_elf_group;Nightblade;;Nachtklinge;;;;;;;;;;;x
```
Comment (arithon) [inline localisation/00_wc_job_titles.csv:236]: Wait, what happened to these titles?

File: localisation/00_zuwc_united_localisation.csv (hunk 8 above)
Comment (BurnedCrusader) [reply localisation/00_zuwc_united_localisation.csv]: No, this was kinda random lol

File: localisation/00_wc_job_titles.csv (hunk 9 above)
Comment (BurnedCrusader) [reply localisation/00_wc_job_titles.csv:236]: They're now work only on culture, not culture group, I don't think I removed them entirely, jsut Github being kinda retarded

Comment (arithon) [reply localisation/00_wc_job_titles.csv:236]: Nope, definitely deleted

File: events/wc_order_events.txt [hunk 10]
```diff
@@ -906,31 +906,131 @@ narrative_event = {
...
+		religion_authority = {
+			modifier = religious_order_formed
+			years = 10
+		}
+		create_character = {
+			random_traits = yes
+			dynasty = none
+			female = yes
+			religion = cult_of_sunwell
+			culture = high_elf
```
Comment (zumbak04) [inline events/wc_order_events.txt:962]:
```suggestion
			culture = ROOT
```

File: events/wc_order_events.txt [hunk 11]
```diff
@@ -906,31 +906,131 @@ narrative_event = {
...
+		}
+		new_character = {
+		    set_defacto_liege = ROOT
+		}
+		create_character = {
+			random_traits = yes
+			dynasty = none
+			female = no
+			religion = cult_of_sunwell
+			culture = high_elf
```
Comment (zumbak04) [inline events/wc_order_events.txt:976]:
```suggestion
			culture = ROOT
```

File: events/wc_order_events.txt [hunk 12]
```diff
@@ -906,31 +906,131 @@ narrative_event = {
...
+			age = 200
+			trait = skilled_tactician
+			trait = creature_high_elf
+			trait = class_paladin_4
+		}
+		new_character = {
+		    set_defacto_liege = ROOT
+		}
```
Comment (zumbak04) [inline events/wc_order_events.txt:970]: Does nothing because he's not a ruler.
```suggestion
```

File: events/wc_order_events.txt [hunk 13]
```diff
@@ -906,31 +906,131 @@ narrative_event = {
...
+			religion = cult_of_sunwell
+			culture = high_elf
+			age = 300
+			trait = creature_high_elf
+			trait = class_paladin_6
+			trait = brilliant_strategist
+		}
+		new_character = {
+		    set_defacto_liege = ROOT
+		}
```
Comment (zumbak04) [inline events/wc_order_events.txt:984]: Does nothing because he's not a ruler.
```suggestion
```

File: events/wc_order_events.txt [hunk 14]
```diff
@@ -906,31 +906,131 @@ narrative_event = {
...
+			
+			activate_title = { title = d_blood_knights_order status = yes }
+			d_blood_knights_order = { grant_title = PREV }
+			set_defacto_liege = ROOT
+			wealth = 600
+		}
+		religion_authority = {
+			modifier = religious_order_formed
+			years = 10
+		}
```
Comment (zumbak04) [inline events/wc_order_events.txt:956]: Can be replaced by a scripted effect.
```suggestion
			#Gives money and moral authority
			holy_order_start_pack_effect = yes
		}
```

File: events/wc_order_events.txt [hunk 15]
```diff
@@ -906,31 +906,136 @@ narrative_event = {
 		name = EVTOPTB_WCORD_406
 	}
 }
+# The Creation of Blood Knights Order
+namespace = WCORD
+narrative_event = {
+	id = WCORD.74
```
Comment (zumbak04) [inline events/wc_order_events.txt:912]: The event ID above is WCORD.406, the event ID below is WCORD.9000. A question, why it''s 74?

File: decisions/wc_order_decisions.txt [hunk 16]
```diff
@@ -250,8 +250,46 @@ decisions = {
...
+		potential = {
+			religion = cult_of_sunwell
+			OR = {
+				has_landed_title = e_quelthalas
+				has_landed_title = k_quelthalas
+			}
+			is_high_elf_trigger = yes
+			
+			d_blood_knights_order = { has_holder = no }
+		}
```
Comment (zumbak04) [inline decisions/wc_order_decisions.txt:269]: I remember there was culture condition...
<details>
<summary>Click to expand</summary>

image

</details>

File: events/wc_order_events.txt [hunk 17]
```diff
@@ -906,31 +906,136 @@ narrative_event = {
 		name = EVTOPTB_WCORD_406
 	}
 }
+# The Creation of Blood Knights Order
+namespace = WCORD
```
Comment (zumbak04) [inline events/wc_order_events.txt:910]: This namespace declared twice.
```suggestion
```

File: interface/wc_decision_icons.gfx [hunk 18]
```diff
@@ -352,4 +352,8 @@ spriteTypes = {
 		name = "GFX_repay_money"
 		texturefile = "gfx\\interface\\decision_icons\\repay_action.tga"
 	}
+	spriteType = {
+		name = "GFX_create_blood_knights_order"
+		texturefile = "gfx\\interface\\decision_icons\\create_blood_knights_order.dds"
```
Comment (zumbak04) [inline interface/wc_decision_icons.gfx:357]: The decision has no icon.
<details>
<summary>Click to expand</summary>

image

</details>

File: decisions/wc_order_decisions.txt (hunk 16 above)
Comment (arithon) [reply decisions/wc_order_decisions.txt:269]: grafik
It was trait-locked but you suggested using trigger

Comment (BurnedCrusader) [reply decisions/wc_order_decisions.txt:269]: Even before trait-lock there was culture condition, but you replaced it with trigger

File: interface/wc_decision_icons.gfx (hunk 18 above)
Comment (BurnedCrusader) [reply interface/wc_decision_icons.gfx:357]: But I put gfx pic in decision_icons and made a clear path for it. Hm

File: decisions/wc_order_decisions.txt (hunk 16 above)
Comment (zumbak04) [reply decisions/wc_order_decisions.txt:269]: > It was trait-locked but you suggested using trigger
It's still race-locked, I'm an elf with Amani culture.

My bad. Meant `is_high_elf_culture_trigger`.
<details>
<summary>Click to expand</summary>

image

</details>

File: events/wc_order_events.txt (hunk 15 above)
Comment (BurnedCrusader) [reply events/wc_order_events.txt]: Just...Because?

Comment (zumbak04) [reply events/wc_order_events.txt]: Can you change it to... WCORD.500 for example? :3

**Pull request merged**